### PR TITLE
Milestone4

### PR DESCRIPTION
**Pull request by AlanSoto31**
Creating calculate module, which is about the app core logic like handling AC, operations(+, *, /, -), =, etc functions. There is another module named operate, where handling the operations of a  basic calculator.

### CODE REVIEW

File: src/logic/operate.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+      r = x * y;
+      break;
+    case '÷':
+      r = x / y;
+      break;
+    case '%':
+      r = x / 100;
+      break;
+    default:
+      r = 'fail';
```
Comment (bellom) [inline src/logic/operate.js]: Job well done but you don't have to do the calculating yourself, that where the use of `big` comes in -> [big js library](https://github.com/favio41/big-js#readme)
example: case division will be -> `total.minus(next)`

File: src/logic/calculate.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
+/* eslint-disable  max-len */
```
Comment (bellom) [inline src/logic/calculate.js]: Inline comment: 
With the ESLint inline comment, you're are indirectly changing the linter's rules by disabling these rules, let's fix the codebase instead.

If you are using VSCode, you can delete the inline comment and hover over the error(configure eslint checks locally). And you will see your options. Click on the more info button that will take you to a GitHub repo where it is explained why you are getting this error. I find this feature very educational. Please, try to learn and implement the necessary rules so you don't have to keep this disabling line on top of each file. I believe you can do this! 💪😇

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi  :clap: 

### Awesome job so far!! :+1: 

#### You have done wonderfully well, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 

# Status: Changes Required! ♻️ 

**Kindly check out my comments below;**

### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Review by Temmarie — APPROVED** [review]
## Status: Approved :heavy_check_mark:

### Hi @AlanSoto31  ! :wave:

#### Thank you for making all of the required changes, your project is approved :clap: :clap: :clap:  :tada:

#### You may merge, good luck with your next projects! :rocket:
#### Happy Coding! :+1:
__Grace Tamara Ekunola __
[GitHub](https://github.com/temmarie) | [LinkedIn](https://www.linkedin.com/in/ekunola-grace/) | [Twitter](https://twitter.com/TemmarieW)

**Pull request merged**